Ex1/main.java

```diff
@@ -59,8 +59,48 @@ public static void main(String[] args) {
                     break;
                 case "11":
                     System.out.println("Goodbye !");
+                    flag = false;
                     break;
             }
         }
     }
+
+    public static void addContact(ArrayList<Contact> phone_book)
+    {
+        Scanner sForAdd = new Scanner (System.in); //open a new scanner for input
+        /*a do-while loop for taking care of invalid phone numbers, using .next() for only
```

Comment:
For proper documentation please refer to this guidelines - 
https://www.oracle.com/il-en/technical-resources/articles/java/javadoc-tool.html